environment/Scripts/activate

```diff
@@ -0,0 +1,66 @@
+# This file must be used with "source bin/activate" *from bash*
```

Comment:
We don't want this file. It should not be used. If you want to automate the virtual environment more. Look at pipenv. That's what all the cool kids are using nowadays https://github.com/pypa/pipenv